### PR TITLE
Update README

### PR DESCRIPTION
**Pull request by HariMahi**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -33,7 +33,17 @@ Anna is part of KWOC : You can talk to us on Zulip or comment on issues to start
...
-
+ # How to install ANNA
+ 
+1. Once you are on the repository page,look for the green “Clone or download” button on the right-hand side of the page.
+https://user-images.githubusercontent.com/29461633/33721594-8bb28f76-db8d-11e7-8e18-6d53aef7e9a2.png
+2. Click the “Clone or download” button and then click “Download ZIP.”
+https://user-images.githubusercontent.com/29461633/33721691-bb2e32a0-db8d-11e7-9452-a73b1b6ec4a8.png
+3. Take the downloaded ZIP file and extract it. We need to give it a permanent home because Chrome will externally load the files (if they get moved/deleted then the extension will stop working).
+4. With the extension downloaded and extracted, we can now install it.In Chrome, navigate to chrome://extensions
+5. Installing the extension is extremely easy. All we need to do is drag-and-drop the folder we extracted onto Chrome’s Extensions page.If the extension is in the “.crx” format, Chrome will ask you to review and approve the permissions before installing.
```
Comment (gauthamzz) [inline README.md]: modify this part a but and add image for this

File: README.md [hunk 2]
```diff
@@ -33,7 +33,17 @@ Anna is part of KWOC : You can talk to us on Zulip or comment on issues to start
   - If I don't understand
   - Close tabs
   
-
+ # How to install ANNA
+ 
+1. Once you are on the repository page,look for the green “Clone or download” button on the right-hand side of the page.
+https://user-images.githubusercontent.com/29461633/33721594-8bb28f76-db8d-11e7-8e18-6d53aef7e9a2.png
```
Comment (gauthamzz) [inline README.md]: Can resize the image

**Review by gauthamzz — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (HariMahi) [reply README.md]: point 5.

File: README.md [hunk 3]
```diff
@@ -33,7 +33,17 @@ Anna is part of KWOC : You can talk to us on Zulip or comment on issues to start
   - If I don't understand
   - Close tabs
   
-
+ # How to install ANNA
+ 
+1. Once you are on the repository page,look for the green “Clone or download” button on the right-hand side of the page.
+![imageedit_3_3326230850](https://user-images.githubusercontent.com/29461633/33730973-a91d3c7c-dba7-11e7-8efe-9d1b9f43980b.png)
```
Comment (gauthamzz) [inline README.md]: I think only this much part is required 
deepinscreenshot_select-area_20171208003554

File: README.md [hunk 4]
```diff
@@ -33,7 +33,17 @@ Anna is part of KWOC : You can talk to us on Zulip or comment on issues to start
...
+2. Click the “Clone or download” button and then click “Download ZIP.”
+![screenshot from 2017-12-07 20_02_51-iloveimg-cropped](https://user-images.githubusercontent.com/29461633/33731004-c02d137e-dba7-11e7-988b-c1527d724896.png)
+3. Take the downloaded ZIP file and extract it. We need to give it a permanent home because Chrome will externally load the files (if they get moved/deleted then the extension will stop working).
```
Comment (gauthamzz) [inline README.md]: from step 3 can you make like this .

3. Using the cloned folder. Go to chrome menu > more tools extensions or chrome:/extensions 
4. Click on load unpacked extension and select the folder and install.
deepinscreenshot_select-area_20171208003514
5. Make sure to give Anna microphone permissions and you are good to go :tada: .

**Review by gauthamzz — CHANGES_REQUESTED** [review]

**Review by gauthamzz — APPROVED** [review]

**Pull request merged**